src/frontends/onnx/tests/CMakeLists.txt

```diff
@@ -159,7 +159,7 @@ endif()
 target_include_directories(ov_onnx_frontend_tests PRIVATE
     $<TARGET_PROPERTY:openvino_onnx_frontend,INCLUDE_DIRECTORIES>)
 
-if(OV_COMPILER_IS_APPLECLANG)
+if(OV_COMPILER_IS_APPLECLANG OR (OV_COMPILER_IS_CLANG AND APPLE))
```

Comment:
Suggest to remove `AND APPLE` and replace with proper version of clang, where the behavior has been changed